### ISSUE TITLE
What did I do wrong?

### ISSUE THREAD

**Issue by EzequielDM** [SoapboxRaceWorld/soapbox-race-core-old#10]
When opening Wildfly, I get this error:

`18:42:39,043 ERROR [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0055: Caught exception during boot: org.jboss.as.controller.persistence.ConfigurationPersistenceException: WFLYCTL0085: Failed
to parse configuration
        at org.jboss.as.controller.persistence.XmlConfigurationPersister.load(XmlConfigurationPersister.java:131)
        at org.jboss.as.server.ServerService.boot(ServerService.java:357)
        at org.jboss.as.controller.AbstractControllerService$1.run(AbstractControllerService.java:299)
        at java.lang.Thread.run(Thread.java:745)
Caused by: javax.xml.stream.XMLStreamException: ParseError at [row,col]:[427,9]
Message: WFLYCTL0069: Duplicate subsystem declaration
        at org.jboss.as.server.parsing.StandaloneXml_4.parseServerProfile(StandaloneXml_4.java:541)
        at org.jboss.as.server.parsing.StandaloneXml_4.readServerElement(StandaloneXml_4.java:242)
        at org.jboss.as.server.parsing.StandaloneXml_4.readElement(StandaloneXml_4.java:141)
        at org.jboss.as.server.parsing.StandaloneXml.readElement(StandaloneXml.java:103)
        at org.jboss.as.server.parsing.StandaloneXml.readElement(StandaloneXml.java:49)
        at org.jboss.staxmapper.XMLMapperImpl.processNested(XMLMapperImpl.java:110)
        at org.jboss.staxmapper.XMLMapperImpl.parseDocument(XMLMapperImpl.java:69)
        at org.jboss.as.controller.persistence.XmlConfigurationPersister.load(XmlConfigurationPersister.java:123)
        ... 3 more

18:43:56,474 FATAL [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0056: Server boot has failed in an unrecoverable manner; exiting. See previous messages for details.`

My config:

https://pastebin.com/EHwru0Ph

**Comment by LeoCodes21** [comment]
You have the Undertow subsystem defined twice. Remove the one that's at line 427.

In layman's terms: Go to line 427, find `<subsystem xmlns="urn:jboss:domain:undertow:3.1">`, and remove everything up until `</subsystem>`. Of course, remove `<subsystem xmlns="urn:jboss:domain:undertow:3.1">` and `</subsystem>` as well.

**Comment by EzequielDM** [comment]
New error, and now bigger

https://pastebin.com/Bq7SShUS

**Comment by LeoCodes21** [comment]
your DB is not running. make sure mysql is started.

**Comment by LeoCodes21** [comment]
No reply after almost 3 weeks, closing. Feel free to re-open if you still require assistance.
